src/main/java/org/jvnet/hudson/test/ToolInstallations.java

```diff
@@ -129,6 +131,33 @@
         return antInstallation;
     }
 
+    /**
+     * Extracts Gradle and configures it.
+     */
+    public static GradleInstallation configureDefaultGradle(TemporaryFolder tmp) throws Exception {
+        GradleInstallation installation;
+        if (System.getenv("GRADLE_HOME") != null) {
+            installation = new GradleInstallation("default", System.getenv("GRADLE_HOME"), JenkinsRule.NO_PROPERTIES);
+        } else {
+            LOGGER.warning("Extracting a copy of Gradle bundled in the test harness. "
+                    + "To avoid a performance hit, set the environment variable GANT_HOME to point to an  Ant installation.");
```

Comment:
`GANT_HOME` or `GRADLE_HOME`? Also, "(...) point to an **Ant installation**."? Shouldn't it say Gradle installation instead?